cmd/crank/render/runtime_docker.go

```diff
@@ -53,6 +54,11 @@ const (
 	// be used for the container. it will also reuse the same container as long as
 	// it is available and also try to restart if it is not running.
 	AnnotationKeyRuntimeNamedContainer = "render.crossplane.io/runtime-docker-name"
+
+	// AnnotationKeyRuntimeEnvironmentVariables sets the environment variables
+	// that will be used for the container. This is helpful to control kpm registry
+	// access to use a different registry.
```

Comment:
Can you update the comment to explain the format please. (I think it's comma separated `VAR=value` pairs right?)

Reply:
Thanks, makes sense. I've added it.